test/AspNet.Security.OAuth.Providers.Tests/Bilibili/BilibiliTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Web;
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace AspNet.Security.OAuth.Bilibili;
+
+public class BilibiliTests(ITestOutputHelper outputHelper) : OAuthTests<BilibiliAuthenticationOptions>(outputHelper)
+{
+    public override string DefaultScheme => BilibiliAuthenticationDefaults.AuthenticationScheme;
+
+    protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+    {
+        builder.AddBilibili(options =>
+        {
+            ConfigureDefaults(builder, options);
+            options.ClientSecret = "ee9ee51ee0ceabdeeeb9459168eeeef7";
```

Comment:
Is the need to specify this rather than use the defaults related to all the hashing?